src/gui_common/MainMenu.tscn

```diff
@@ -324,6 +325,17 @@ __meta__ = {
 
 [node name="GUIAnimations" type="AnimationPlayer" parent="."]
 anims/MenuSlide = SubResource( 1 )
+
+[node name="GLES2 Popup" type="ConfirmationDialog" parent="."]
+margin_right = 515.0
+margin_bottom = 108.0
+theme = ExtResource( 2 )
+window_title = "GLES2 Mode Warning"
+dialog_text = "You are running Thrive with GLES2. This is severely untested and is likely to cause issues."
```

Comment:
Advice on trying to update video drivers and force nvidia or amd graphics to be used, would be nice to include here.